src/pages/templates/forms/newsletter/emailOnly.tsx

```diff
@@ -0,0 +1,94 @@
+import { FormEvent, ChangeEvent, useState } from "react";
+import {
+  Stack,
+  FormControl,
+  Input,
+  Button,
+  useColorModeValue,
+  Flex,
+  Text,
+} from "@chakra-ui/react";
+import { CheckIcon } from "@chakra-ui/icons";
+
+export default function EmailOnly() {
+  const [email, setEmail] = useState("");
+  const [state, setState] = useState<"initial" | "submitting" | "success">(
+    "initial"
+  );
+  const [error, setError] = useState(false);
+
+  return (
+    <Flex w="100%" my={6} mx={2}>
+      <Flex
+        w={["100%", "95%", "80%", "70%"]}
```

Comment:
We prefer to use the object syntax for responsive values for a better readability. https://chakra-ui.com/docs/features/responsive-styles#the-object-syntax

Reply:
makes sense, I've changed it to the object syntax